examples/darwin-framework-tool/commands/common/CHIPCommandBridge.h

```diff
@@ -41,6 +41,8 @@ class CHIPCommandBridge : public Command {
             "Sets the commissioner node ID of the given "
             "commissioner-name. Interactive mode will only set a single commissioner on the inital command. "
             "The commissioner node ID will be persisted until a different one is specified.");
+        AddArgument("commissioner-shared-storage", 0, 1, &mCommissionerSharedStorage,
+            "Use a shared storage instance instead of individual storage for each commissioner. Defaults is true.");
```

Comment:
Followup: After this lands, we should consider changing the default to false.

Reply:
> Followup: After this lands, we should consider changing the default to false.

Agreed. I initially thought this was the desired default but wasn’t entirely sure, so I kept it set to `true`. Let’s handle changing the default to `false` in a follow-up PR as you suggest, focusing specifically on any potential CI issues that may come up.